### PR TITLE
Nav styling

### PR DESCRIPTION
**Pull request by geet084**

### CODE REVIEW

File: src/css/App.css [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 .app-title-area {
   height: 175px;
-  opacity: 0.7;
+  /* opacity: 0.7; */
```
Comment (b3an5) [inline src/css/App.css]: may want to delete this

File: src/css/App.css [hunk 2]
```diff
@@ -47,6 +47,12 @@
   max-width: 100%;
 }
 
+.nav-area {
+  background: url('../images/nav.jpg');
+  height: 120px;
+  /* opacity: .2; */
```
Comment (b3an5) [inline src/css/App.css]: and this

File: src/css/App.css (hunk 1 above)
Comment (geet084) [reply src/css/App.css]: touche!

File: src/Nav.js [hunk 3]
```diff
@@ -38,29 +27,26 @@ export default class Nav extends Component {
...
 
           <select id="division-select"
                   value={this.state.selectedValue}
-                  onChange={this.handleChange} >
-            <option value="all">--Show All--</option>
-            <option value="AFC">--AFC Division--</option>
+                  onChange={this.setSelection} >
+            <option value="init" disabled>Select a Division: </option>
+            <option value="all">Show All</option>
+            <option disabled></option>
```
Comment (ericfitzsimons451) [inline src/Nav.js]: cool way to get a space in the dropdown

File: src/css/App.css (hunk 2 above)
Comment (geet084) [reply src/css/App.css]: fixed!

**Pull request merged**